app/views/shopify_app/sessions/request_storage_access.html.erb

```diff
@@ -5,6 +5,11 @@
   <meta name="viewport" content="width=device-width, initial-scale=1" />
   <base target="_top">
   <title>Redirecting…</title>
+  <script src="https://unpkg.com/@shopify/app-bridge@^1"></script>
+  <script>
+    window.apiKey = "<%= ShopifyApp.configuration.api_key %>";
+    window.shopOrigin = "https://<%= current_shopify_domain %>";
```

Comment:
If the only thing consuming this is App Bridge, and it doesn't want the `https://`, maybe we should drop it